test-cases/nemesis/longevity-5gb-1h-BootstrapStreamingErrMonkey.yaml

```diff
@@ -0,0 +1,37 @@
+test_duration: 90
```

Comment:
Please use the hydra command to generate the nemesis pipeline, don't copy it by hand

Reply:
@fruch your comment is for the pipeline, right>
Not for the yaml.

Reply:
Removed this commit